tg_bot.py

```diff
@@ -22,6 +23,9 @@ def __init__(self, twitter_api, dirpath, level, auto_retweet):
         self.nw_logger = nawab_logger.Nawab_Logging(dirpath, level)
         self.auto_retweet = auto_retweet
 
+    def line_numb(self):
```

Comment:
Please use complete words as function names. According to weather, it is a public or private function, you can prepend nawab_* infront.

Reply:
okay